src/Microsoft.VisualStudio.Composition.AppHost/CreateComposition.cs

```diff
@@ -44,6 +44,16 @@ public class CreateComposition : AppDomainIsolatedTask, ICancelableTask
         /// </summary>
         private readonly List<string> catalogAssemblyPaths = new List<string>();
 
+        /// <summary>
+        /// Gets the assemblies that have been logged as to their locations already.
```

Comment:
This might be clearer if you said "Gets the assemblies that have been found and logged."

Reply:
Done